packages/evm/contracts/court/controller/Controller.sol

```diff
@@ -234,6 +260,16 @@ contract Controller is IsContract, Modules, CourtClock, CourtConfig {
         _setModulesGovernor(ZERO_ADDRESS);
     }
 
+    /**
+    * @notice Set custom function `_sig` for `_target`
+    * @param _sig Signature of the function to be set
+    * @param _target Address of the target implementation to be registered for the given signature
+    */
+    function setCustomFunction(bytes4 _sig, address _target) external onlyModulesGovernor {
```

Comment:
Should we add this to the docs, and perhaps explain our intended use case?

Reply:
Addressed in 53fa8d05958a711bd11aee4b3d8de5d7fece9e57